src/components/backupWallet/ExportWalletData.js

```diff
@@ -0,0 +1,151 @@
+// @flow
+
+// libraries
+import React, { useMemo } from 'react'
+import { Image, TouchableOpacity, View } from 'react-native'
+
+// components
+import Wrapper from '../common/layout/Wrapper'
+import { CustomButton, Icon, Section } from '../common'
+import NavBar from '../appNavigation/NavBar'
+
+// hooks
+import useOnPress from '../../lib/hooks/useOnPress'
+import useClipboard from '../../lib/hooks/useClipboard'
+
+// utils
+import { withStyles } from '../../lib/styles'
+import { getDesignRelativeHeight } from '../../lib/utils/sizes'
+import GoodWallet from '../../lib/wallet/GoodWallet'
+import GDStore from '../../lib/undux/GDStore'
+import config from '../../config/config'
+
+// assets
+import unknownProfile from '../../assets/unknownProfile.svg'
+
+// getting the privateKey of GD wallet address - which index is 0
+const privateKey = GoodWallet.wallet.eth.accounts.wallet[0].privateKey
+const web3ProviderUrl = config.ethereum[GoodWallet.networkId].httpWeb3provider
+
+type BackupWalletProps = {
+  styles: {},
+  theme: {},
+  screenProps: any,
+}
+
+const BorderedBox = ({ styles, theme, avatarSource, title, content, copyButtonText }) => {
+  const { setString } = useClipboard()
+  const copyToClipboard = useOnPress(() => setString(content), [setString, content])
```

Comment:
please don't forget to add usePermissions once #1790 will be merged on master

Reply:
not required here - it's not read operation, but write.

Reply:
okay, could you please look at #1790 and remove usePermission where only write to clipboard is performed ? (e.g. copybutton) ?

Reply:
Pushed a fix for #1966